infrastructure/modules/serverless/data.tf

```diff
@@ -15,6 +15,5 @@
 # tflint-ignore: terraform_unused_declarations
 data "aws_kms_alias" "check_existing_kms" {
   count = var.env == "prod" ? 1 : 0
-  name = "alias/${var.namespace}-kms-key"
-
+  name  = "alias/${var.project}-kms-key"
```

Comment:
in prod env, kms created with project variable but in data.tf it try to search with namespace variable and raise error. So I changed to project variable.